doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -271,7 +271,20 @@ pimcore:
 - Moved implementation of `PimcoreBundleAdminSupportInterface` from `AbstractPimcoreBundle` to bundle classes.
     Moved `getJsPaths`, `getCssPaths`, `getEditmodeJsPaths` and `getEditmodeCssPaths` from `AbstractPimcoreBundle` to `BundleAdminSupportTrait`.
 - [Cache] Responses containing a header `Cache-Control: no-cache`, `Cache-Control: private` or `Cache-Control: no-store` will no longer be cached by the full page cache.
-
+- [Documents] Configuration of document types has changed. Now it is possible to change the navigational behavior and more of each document type in the configuration.
+```yaml
+    documents:
+        type_definitions:
+            map:
+                page:
+                    class: \Pimcore\Model\Document\Page
```

Comment:
need to change the example under 
https://github.com/pimcore/pimcore/blob/14268-enhance-document-configs/doc/20_Extending_Pimcore/13_Bundle_Developers_Guide/08_Adding_Document_Types.md#2-register-the-document-on-the-document-type-map